src/Platform/Microsoft.Testing.Platform/OutputDevice/TerminalOutputDevice.cs

```diff
@@ -93,7 +93,7 @@ public TerminalOutputDevice(
             _targetFramework = TargetFrameworkParser.GetShortTargetFramework(Assembly.GetEntryAssembly()?.GetCustomAttribute<TargetFrameworkAttribute>()?.FrameworkDisplayName) ?? _runtimeFramework;
         }
 
-        _assemblyName = testApplicationModuleInfo.GetCurrentTestApplicationFullPath();
+        _assemblyName = testApplicationModuleInfo.TryGetCurrentTestApplicationFullPath() ?? testApplicationModuleInfo.TryGetAssemblyName() ?? "unknown assembly name";
```

Comment:
This is used for display only. So, if we don't have a full path, we can try assembly name, or fallback to unknown assembly